### PR TITLE
Add MySQL

### PR DESCRIPTION
**Pull request by jycouet**
It's already working 🥳
Needs now a few _"real"_ project to finish the last queries + test types.

- [x] getTablesInfo
- [x] getTableColumnInfo 
- [ ] getUniqueInfo
- [ ] getForeignKeys
- [ ] getEnumDef

### CODE REVIEW

File: src/db/DbMySQL.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+			// Knex client configuration for MySQL
+			client: "mysql2",
+			connection: connectionString,
+		});
+	}
+
+	async getTablesInfo() {
+		const result = await this.knex!.knex.raw("SHOW TABLES");
+		// eslint-disable-next-line
+		return result[0].map((row: any) => {
```
Comment (Yedidyar) [inline src/db/DbMySQL.ts:20]: What the type of result?

**Comment by Yedidyar** [comment]
Make sure that you implement the todo

File: src/db/DbMySQL.ts (hunk 1 above)
Comment (jycouet) [reply src/db/DbMySQL.ts:20]: A Json

**Comment by Yedidyar** [comment]
@jycouet  I am closing for now if you want to add support  mysql please create new PR with unit tests and e2e

**Pull request closed without merging**